components/director/internal/domain/labeldef/service_test.go

```diff
@@ -140,19 +141,232 @@ func TestServiceList(t *testing.T) {
 		defer mockRepository.AssertExpectations(t)
 		ctx := context.TODO()
 		mockRepository.On("List", ctx, "tenant").Return(nil, errors.New("some error"))
-		sut := labeldef.NewService(mockRepository, nil)
+		sut := labeldef.NewService(mockRepository, nil, nil)
 		// WHEN
 		_, err := sut.List(ctx, "tenant")
 		// THEN
 		require.EqualError(t, err, "while fetching Label Definitions: some error")
 	})
 }
 
+func TestServiceUpdate(t *testing.T) {
+	tenant := "tenant"
+	key := "firstName"
+
+	t.Run("success", func(t *testing.T) {
```

Comment:
add test case that new schema is nil

Reply:
done

Reply:
where???

Reply:
`returns error if Label Definition schema for update is nil` inside `labeldef/service_test.go`

Reply:
`label definition schema for update is nil` inside `labeldef/service_test.go`